DESCRIPTION.md

```diff
@@ -9,7 +9,13 @@ Source code is also available at:
 
 # Release Notes
 
-- v1.7.0(November 22, 2024)
+- (Unreleased)
+  - Added `force_div_is_floordiv` flag to override `div_is_floordiv` new default value `False` in `SnowflakeDialect`.
+    - With the flag in `False`, the `/` division operator will be treated as a float division and `//` as a floor division.
```

Comment:
This is not how it works.

This flag is a signal from the dialect towards sqlalchemy that describes the behavior of this sql implementation (Snowflake). So if `select 4/5` returns 0, `div_is_floordiv` should be set to true. If it returns `0.8`, this flag must be set to false.

So the previous value was just a bug, which should be fixed. There is no need to make this configurable.

Reply:
Hi @jochenott 

We agree that the correct value for the flag `div_is_floordiv` in Snowflake should be False. However, current users expect the `/ ` and `//` operators to work as true divisions, although `/ `should behave as a true division and `//`  as a floor division. That's why the behavioral change will be introduced as a flag, which will be removed in a future major release.

Thank you for pointing out this code issue and your contribution!!!